src/Core/Grid/Definition/Factory/Security/Session/CustomerGridDefinitionFactory.php

```diff
@@ -0,0 +1,274 @@
+            )
+            ->add(
+                (new SimpleGridAction('common_export_sql_manager'))
+                    ->setName($this->trans('Export to SQL Manager', [], 'Admin.Actions'))
+                    ->setIcon('storage')
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters(): FilterCollectionInterface
+    {
+        return (new FilterCollection())
+            ->add(
+                (new Filter('id_customer_session', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'size' => 'small',
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('id_customer_session')
+            )
+            ->add(
+                (new Filter('id_customer', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'size' => 'small',
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('id_customer')
+            )
+            ->add(
+                (new Filter('firstname', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('First name', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('firstname')
+            )
+            ->add(
+                (new Filter('lastname', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Last name', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('lastname')
+            )
+            ->add(
+                (new Filter('email', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Email', [], 'Admin.Global'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('email')
+            )
+            ->add(
+                (new Filter('actions', SearchAndResetType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'data-url' => $this->resetActionUrl,
+                            'data-redirect' => $this->redirectionUrl,
+                        ],
+                    ])
```

Comment:
New grids look more like this:

```suggestion
                (new Filter('actions', SearchAndResetType::class))
                    ->setTypeOptions([
                        'attr' => [
                            'reset_route' => 'admin_common_reset_search_by_filter_id',
                        'reset_route_params' => [
                            'filterId' => self::GRID_ID,
                        ],
                        'redirect_route' => 'admin_security_sessions_customer_search',
                        ],
                    ])
```

Especially since you use a grid with `filterId` and the common search controller (so it is used as a persistence key) but you set a reset controller based on controller/action (which is the old way) So I'm not sure the reset works correctly